src/coordinax/_src/vectors/checks.py

```diff
@@ -82,3 +78,179 @@ def check_polar_range(
         xp.any(xp.logical_or((polar < _l), (polar > _u))),
         "The inclination angle must be in the range [0, pi].",
     )
+
+
+def check_non_negative(x: AbstractQuantity, /, name: str = "") -> AbstractQuantity:
```

Comment:
Should the 'name' args maybe be kwargs? What do you think?

```suggestion
def check_non_negative(x: AbstractQuantity, /, *, name: str = "") -> AbstractQuantity:
```